### ISSUE TITLE
TeX-compatible export

### ISSUE THREAD

**Issue by derTimme** [simonsmiley/iliasCorrector#4]
Remark export should support tex-export. That is, prefixing the remark field in the exported `points.csv` with `%tex\n` (note the line break!).

The actual question is: When to do this? Maybe a setting for each exercise? Or an automatic detection (backslashes in the text, `$`s, empty line followed a non-empty one, ...) for each single submission?

Open for discussion, the implementation shouldn't be a problem, I just want to produce a solution suitable for everyone, not just for me.

**Comment by simonsmiley** [comment]
Hmm... A global setting would be the easiest to implement. 
A setting pee exercise would require changing how exercises are set up, but it's doable. 
The other option that comes to mind is searching for specific Tex commands in the remarks. 
I would only do this in the export

**Comment by derTimme** [comment]
Yes, but a global setting could endanger backwards compatibility.

My personal preference would be automatic detection as described in my initial comment.

And of course, this only makes sense for the actual export - otherwise one would have these `%tex` lines all over the place in the remarks.
